src/stylesheets/elements/form-controls/_combo-box.scss

```diff
@@ -42,25 +42,26 @@ button.usa-combo-box__clear-input {
 .usa-combo-box__clear-input {
   background-color: transparent;
   background-position: center;
-  background-size: auto units(1.5);
```

Comment:
We might need to tweak the dimensions of the inputs.

Reply:
Fixed